knowledge/college/mascots/reveille/qna.yaml

```diff
@@ -0,0 +1,27 @@
+created_by: headbrad
+domain: college_mascot
+seed_examples:
+  - answer: |
+      There have been ten Reveille's in the history of Texas A&M with the first Reveille I joining in 1931.
+    question: How many Reveille Dogs have there been in the history of Texas A&M?
+  - answer: |
+      Reveille is most commonly recognized as a purebred Rough Collie. However, Reveille I was a mixed-breed dog and Reveille II was Shetland Sheepdog.
+    question: What type of dog is Reveille?
```

Comment:
In an effort to distinguish the A&M Reveille dog from other types of Reveille, maybe try "what type of dog is the A&M Reveille dog" This would apply to your other questions as well.